reposerver/repository/repository.go

```diff
@@ -227,7 +227,7 @@ func helmTemplate(appPath string, q *apiclient.ManifestRequest) ([]*unstructured
 		}
 		templateOpts.Values = appHelm.ValueFiles
 		if appHelm.Values != "" {
-			file, err := ioutil.TempFile("", "values-*.yaml")
+			file, err := ioutil.TempFile(appPath, "values-*.yaml")
```

Comment:
As part of this change, all values stemming from `Helm.Values` will now have to be saved in a temp file within the application path.

Reply:
clever